dts/msm8916-samsung-r02.dts

```diff
@@ -0,0 +1,22 @@
+// SPDX-License-Identifier: GPL-2.0-only
+
+/dts-v1/;
+
+/include/ "msm8916.dtsi"
+
+/ {
+	// This is used by the bootloader to find the correct DTB
+	qcom,msm-id = <206 0>;
+	qcom,board-id = <0xCE08FF01 2>;
+
+	gprimelte {
```

Comment:
This is too generic. The Android device name appears to be `gprimeltecan`, see

https://github.com/jaredrummler/AndroidDeviceNames/blob/e23b73dbb81be6cb64dfa541a3e93800ee26b185/json/manufacturers/SAMSUNG.json#L3664-L3669